# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/daja77/pil/freetype_tweaking.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./_imagingft.c.vanilla	2005-03-01 15:36:02.557307912 +0100
+++ ./_imagingft.c	2005-03-01 15:36:12.027868168 +0100
@@ -15,6 +15,7 @@
 #include "Python.h"
 #include "Imaging.h"
 
+#include <ft2build.h>
 #include <freetype/freetype.h>
 
 #if PY_VERSION_HEX < 0x01060000
--- ./setup.py.vanilla	2005-03-01 15:10:13.111859424 +0100
+++ ./setup.py	2005-03-01 15:29:12.846593408 +0100
@@ -25,7 +25,7 @@
 # include files in the directories below.  tweak as necessary.
 JPEGDIR = "../../kits/jpeg-6b"
 ZLIBDIR = "../../kits/zlib-1.1.4"
-FREETYPEDIR = "../../kits/freetype-2.0"
+FREETYPEDIR = ""
 
 # on Windows, the following is used to control how and where to search
 # for Tcl/Tk files.  None enables automatic searching; to override, set
